### PR TITLE
Param layout update

### PR DESCRIPTION
**Pull request by BryanRumsey**

### CODE REVIEW

File: client/views/edit-parameter.js [hunk 1]
```diff
@@ -96,6 +102,10 @@ module.exports = View.extend({
       this.model.collection.trigger('remove')
     }
   },
+  updateViewer: function () {
+    console.log(this.parent)
```
Comment (jtcooper10) [inline client/views/edit-parameter.js:106]: Debugging log left in

File: client/views/parameters-editor.js [hunk 2]
```diff
@@ -30,14 +30,13 @@ module.exports = View.extend({
   template: template,
   events: {
     'click [data-hook=add-parameter]' : 'addParameter',
-    'click [data-hook=save-parameters]' : 'switchToViewMode',
     'click [data-hook=collapse]' : 'changeCollapseButtonText',
   },
   initialize: function (attrs, options) {
     var self = this;
     View.prototype.initialize.apply(this, arguments);
+    this.readOnly = attrs.readOnly ? attrs.readOnly : false;
```
Comment (jtcooper10) [inline client/views/parameters-editor.js:38]: `attrs.readOnly ? attrs.readOnly : false` is always equivalent to the value of `attrs.readOnly`

File: client/views/parameters-editor.js [hunk 3]
```diff
@@ -88,6 +96,24 @@ module.exports = View.extend({
...
+    this.viewParameterView = this.renderCollection(
+      this.collection,
+      EditParameterView,
+      this.queryByHook('view-parameter-list'),
+      options
+    );
+  },
   addParameter: function () {
     this.collection.addParameter();
     $(document).ready(function () {
```
Comment (jtcooper10) [inline client/views/parameters-editor.js:119]: (Unrelated to PR) jQuery's `$(document).ready()` function appears to be deprecated

**Review by jtcooper10 — APPROVED** [review]

File: client/views/parameters-editor.js (hunk 2 above)
Comment (BryanRumsey) [reply client/views/parameters-editor.js:38]: We need to check if `attrs.readOnly` is defined

File: client/views/edit-parameter.js [hunk 4]
```diff
@@ -43,9 +44,14 @@ module.exports = View.extend({
   initialize: function (attrs, options) {
     View.prototype.initialize.apply(this, arguments);
     this.previousName = this.model.name
+    this.viewMode = attrs.viewMode ? attrs.viewMode : false;
```
Comment (ethangreen-dev) [inline client/views/edit-parameter.js:47]: If `attr.viewMode` is a bool then you don't need the ternary operator.

**Review by ethangreen-dev — APPROVED** [review]

File: client/views/parameters-editor.js (hunk 3 above)
Comment (BryanRumsey) [reply client/views/parameters-editor.js:119]: fixed

**Pull request merged**